guides/common/modules/proc_configuring-kerberos-authentication-for-remote-execution.adoc

```diff
@@ -9,7 +9,7 @@ You can use Kerberos authentication to establish an SSH connection for remote ex
 * Enroll {ProjectServer} on the Kerberos server
 * Enroll the {Project} target host on the Kerberos server
 * Configure and initialize a Kerberos user account for remote execution
-* Ensure that the foreman-proxy user on {Project} has a valid Kerberos ticket granting ticket
+* Ensure that the `foreman-proxy` user on {Project} has a valid Kerberos ticket granting ticket
```

Comment:
The change looks fine. I was wondering whether "valid Kerberos ticket granting ticket" was correct or a typo.